Eastern-Research-Group/expert-query
# Feature/251 restructure api endpoints

## Related Issues:
* [EQ-251](https://jira.epa.gov/browse/EQ-251)

## Main Changes:
* Updated the structure of the api endpoints.
* Fixed syntax errors in the `swagger/attains.json` file.
* Moved the `app/server` environment checking logic to a shared location like what we do for the etl.
* Added an `api/openapi` endpoint to support swagger. This one is cors enabled.

## Steps To Test:
1. Run the etl and verify it succeeds
2. Start the app 
3. Verify all of the features work:
  * Filtering values in Organization ID and Assessment Unit ID dropdowns
  * Running queries
  * National Downloads page
  * API Documentation
4. Navigate to the following urls and verify they work (should just say `"status": "UP"`)
  * http://localhost:9090/api/health
  * http://localhost:9090/api/health/etlGlossary
  * http://localhost:9090/api/attains/health/etlDatabase
  * http://localhost:9090/api/attains/health/etlDomainValues 
5. Navigate to http://localhost:3000/api-documentation
6. Verify that all of the servers show up in the dropdown
7. In the `app/server/app/routes/api.js` file, change line 98 from `if (environment.isProduction) {` to `if (!environment.isProduction) {`
8. Reload the api documentation page
9. Verify that only the production server is available in the dropdown
